website/src/content/development/testing.mdx

```diff
@@ -298,7 +298,7 @@ The `.env` file should be git ignored. On CI, you can define environment variabl
 In the `cypress/plugins/index.js` file, you need to configure the `task` for Custom Applications and to load the environment variables:
 
 ```js title="cypress/plugins/index.js"
-const customApplications = require('@commercetools-frontend/cypress/task');
+const { customApplicationConfig } = require('@commercetools-frontend/cypress/task');
```

Comment:
@kark will be a breaking change for users already using this API? If so, we should include a release note.

Reply:
Hi @kark, looks like the legacy export will solve the breaking change. Can you confirm this with a manual test? if it's good then we don't need a release note.

Reply:
Thanks, of course I tested it manually.

Steps to follow test it out:

1. Checkout the `SHIELD-993-custom-view-e2e-testing-docs` and apply changes:
<img width="693" alt="Screenshot 2024-01-18 at 13 56 24" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/fe547e8e-cdcc-466f-954f-5df95ec03135">

2. Build monorepo packages
3. Run `pnpm template-custom-application-starter:start:prod:local` in one terminal
4. Run `pnpm test:e2e:template-custom-application-starter` in another terminal

Error encountered without the legacy export:
<img width="761" alt="Screenshot 2024-01-18 at 14 26 43" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/43ada368-07e6-45e2-b284-c2dea401efe8">

With the legacy export the tests work correctly.

Reply:
Yayy! Thanks @kark for going all the way 🙌🏼